### PR TITLE
sync :has() doc with English version and translate into Simplified Chinese

### PR DESCRIPTION
**Pull request by liunnn1994**
The old document is out of date, some of this content is already wrong.

### CODE REVIEW

File: files/zh-cn/web/css/_colon_has/index.md [hunk 1]
```diff
@@ -1,35 +1,44 @@
 ---
-title: ':has'
+title: ':has()'
 slug: Web/CSS/:has
+tags:
```
Comment (awxiaoxian2020) [inline files/zh-cn/web/css/_colon_has/index.md:4]: 请参照[翻译指南](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#元数据)移除 `tags` 等元数据。

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/css/_colon_has/index.md [hunk 2]
```diff
@@ -40,3 +49,8 @@ h1:has(+ p)
 ## 浏览器兼容性
 
 {{Compat}}
+
+## 相关链接
```
Comment (awxiaoxian2020) [inline files/zh-cn/web/css/_colon_has/index.md:53]:
```suggestion
## 参见
```

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/css/_colon_has/index.md (hunk 1 above)
Comment (asdjgfr) [reply files/zh-cn/web/css/_colon_has/index.md]: 移除了 `tags` 以及 `browser-compat`

File: files/zh-cn/web/css/_colon_has/index.md (hunk 2 above)
Comment (asdjgfr) [reply files/zh-cn/web/css/_colon_has/index.md]: 已修改并 `squash`.

**Review by yin1999 — APPROVED** [review]
Thanks @asdjgfr @awxiaoxian2020

**Pull request merged**